Fix the anchors in the contribution guide

chasenlehara opened the pull request:
Related to https://github.com/canjs/bit-docs-html-canjs/issues/208

chasenlehara committed:
Fix the anchors in the contribution guide

Related to https://github.com/canjs/bit-docs-html-canjs/issues/208

phillipskevin commented:
This is broken because of another change. It will be fixed by https://github.com/donejs/donejs/pull/844.

matthewp committed:
Merge pull request #867 from donejs/major

DoneJS 1.0

matthewp committed:
Update main test file for PMO

This updates the main test file for PMO which reflects that the
functional tests are no longer in a separate file.

This change is a result of
https://github.com/donejs/generator-donejs/commit/2e9a7e92559761b1fa8aaf11dd0c20ce629c5ac3

matthewp committed:
Update the location of the test.html

matthewp committed:
Only run PMO guide in latest node version

matthewp committed:
Merge pull request #873 from donejs/master-master

Update tests for new generated app test structure

matthewp committed:
1.0.0
